DiagnosticsExtension/Controllers/ConnectionStringValidationController.cs

```diff
@@ -7,41 +7,46 @@
 
 using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Net;
 using System.Net.Http;
 using System.Net.Sockets;
 using System.Text;
 using System.Threading.Tasks;
+using System.Web;
 using System.Web.Http;
+using DiagnosticsExtension.Models;
 using DiagnosticsExtension.Models.ConnectionStringValidator;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace DiagnosticsExtension.Controllers
 {
-    /// <summary>
-    /// Worker instances are running an udp echo server on port 30000. This controller is for checking the connection between target 
-    /// worker instance by pinging and checking the echoed result.
-    /// </summary>
     [RoutePrefix("api/connectionstringvalidation")]
     public class ConnectionStringValidationController : ApiController
     {
         [HttpGet]
         [Route("validate")]
         public async Task<HttpResponseMessage> Validate(string connStr, int? typeId = null)
```

Comment:
Would be convenient to take the typeId in as a string so callers do not need to know the enum mapping.   Should make manual validation in the future easier as well.  Can use Enum.Parse() in the function

Reply:
done